app-text/sgml-common/sgml-common-0.6.3-r6.ebuild

```diff
@@ -0,0 +1,85 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="6"
+
+inherit eutils prefix
```

Comment:
Do you really need eutils in EAPI=6?

Reply:
Nope it's not needed. Removed it

Reply:
https://github.com/gentoo/gentoo/pull/6297/commits/3b60c5ef7b9d721887d9ee7310a0cb96abdc245c